src/test/java/org/folio/services/ftp/FtpUploadServiceTest.java

```diff
@@ -190,4 +194,27 @@ public void testFailedUpload(VertxTestContext vertxTestContext) throws URISyntax
     vertxTestContext.assertFailure(future)
       .onComplete(result -> vertxTestContext.completeNow());
   }
+
```

Comment:
please add test when credentials.username or credentials.password is empty

Reply:
yes, working on it...